specfile/changelog.py

```diff
@@ -350,13 +394,24 @@ def extract_following_lines(content: List[str]) -> List[str]:
         predecessor = []
         header = None
         content: List[str] = []
-        for line in section:
-            if line.startswith("*"):
+
+        for i, line in enumerate(section):
+            if line == _OPENSUSE_CHANGELOG_SEPARATOR:
+                continue
+
+            if line.startswith("*") or (
+                i >= 1 and section[i - 1] == _OPENSUSE_CHANGELOG_SEPARATOR
```

Comment:
The same condition is used twice, it would make sense to store the result into a variable.